pages/desktop/submit.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.desktop.base import Base
+
+
+class Submit(Base):
+
+    _upload_addon_locator = (By.ID, "upload-addon")
+    _upload_progress_bar_success_locator = (By.CSS_SELECTOR, "#upload-file>.upload-status>.bar-success")
+    _misc_category_locator = (By.CSS_SELECTOR, ".addon-misc-category.checkbox-choices>li>label")
+    _mpl_license_locator = (By.CSS_SELECTOR, "#id_builtin>li>label")
+    _review_button_locator = (By.CSS_SELECTOR, ".submit-buttons>button")
+    _done_next_steps_locator = (By.CSS_SELECTOR, ".done-next-steps")
+    _continue_to_step_3_locator = (By.ID, "submit-upload-file-finish")
+    _continue_to_step_4_locator = (By.CSS_SELECTOR, ".submission-buttons.addon-submission-field>button")
+    _continue_to_step_5_locator = (By.CSS_SELECTOR, ".edit-media-button.addon-submission-field>button")
+    _continue_to_step_6_locator = (By.CSS_SELECTOR, ".submission-buttons.addon-submission-field>button")
```

Comment:
@davehunt Should we have this as a separate locator though, just to ensure readability so we're not using `_continue_to_step_four_locator` in the `continue_to_step_six` method?